src/com/dirj/DirJ.java

```diff
@@ -3,38 +3,73 @@
 import java.io.File;
 import java.text.NumberFormat;
 import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.Locale;
+import java.util.*;
 
 public class DirJ {
     private static final String DIR = "  <DIR>";
+    private static boolean onlyFiles = false;
+    private static boolean lowerCase = false;
+    private static boolean helpOption = false;
+
     public static void main(String[] args) {
+        for (String arg : args) {
+            switch (arg) {
+                case "/B":
+                    onlyFiles = true;
+                    break;
+                case "/L":
+                    lowerCase = true;
+                    break;
+                case "/?":
+                    helpOption = true;
+                    break;
+                default:
+                    System.out.println("Unknown command");
```

Comment:
Minor; it's always a good idea to suggest how to get proper help for your utility if someone used unknown argument. E.g. `"Unknown argument '" + arg + "'. Use '/?' argument for help."`